.pre-commit-config.yaml

```diff
@@ -44,11 +44,11 @@ repos:
           )$
 
   - repo: https://github.com/golangci/golangci-lint
-    rev: v1.51.0
+    rev: v1.54.2
     hooks:
       - id: golangci-lint
         name: Lint go files
-        args: ['--concurrency', '-1'] # Run with all CPUs
+        args: ['--concurrency', '-1', '-v'] # Run with all CPUs
```

Comment:
I don't think the `-v` argument is necessary here - looking at the MINT PR, I think it was just added there for debugging.

Reply:
Yeah, we'd determined that only logs show for the individual linters show up on fail (or if you make `pre-commit` run in verbose mode) so this just lets us see more in the case of a failure, so I thought to add it here as well!

Reply:
Got it (after discussion) - these verbose logs from `golangci-lint` will be swallowed by `pre-commit` if the check passes, they'll only be output if the check fails. This should be fine, then.